modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -81,18 +72,44 @@ class JournalPortlet extends PortletBase {
 		this._setupSidebar();
 	}
 
-	/**
-	 * @inheritDoc
-	 */
-	created() {
-		this._eventHandler = new EventHandler();
-	}
-
 	/**
 	 * @inheritDoc
 	 */
 	detached() {
-		this._eventHandler.removeAllListeners();
+		this._buttonClickUpdateAction.dispose();
+
+		const form = this._getInputByName(this.ns('fm1'));
+
+		if (form) {
+			form.removeEventListener('submit', this._onFormSubmit.bind(this));
```

Comment:
You get the idea by now. I'm going to stop commenting here because I'm spamming too much.

Reply:
✔️  yea, thanks @wincent I'll try and do this today.

Reply:
Hey @kresimir-coko focus on the MultiSelect, I get this one ;)

Reply:
Alright, thanks @carloslancha

Reply:
@wincent yup, I noticed that and I'm working on it, thx!!